src/v/cluster/data_migration_backend.h

```diff
@@ -65,7 +65,7 @@ class backend {
       = chunked_hash_map<model::topic_namespace, topic_reconciliation_state>;
     struct migration_reconciliation_state {
         explicit migration_reconciliation_state(work_scope scope)
-          : scope(scope){};
+          : scope(scope) {};
```

Comment:
nit: not from this PR but extraneous semicolon

Reply:
ha, fun. will remove in separate PR.